esp-box-3/esp-box-3.c

```diff
@@ -626,3 +626,24 @@ esp_err_t bsp_iot_button_create(button_handle_t btn_array[], int *btn_cnt, int b
     }
     return ret;
 }
+
+bsp_imu_t bsp_imu_init(void)
+{
+    /* Initilize I2C */
+    BSP_ERROR_CHECK_RETURN_ERR(bsp_i2c_init());
+
+    /* Initialize ICM42670 */
+    icm42670_handle_t handle = icm42670_create(BSP_I2C_NUM, ICM42670_I2C_ADDRESS);
+    assert(handle);
+
+    /* Configuration of the acceleremeter and gyroscope */
+    const icm42670_cfg_t imu_cfg = {
+        .acce_fs = ACCE_FS_2G,
+        .acce_odr = ACCE_ODR_400HZ,
+        .gyro_fs = GYRO_FS_2000DPS,
+        .gyro_odr = GYRO_ODR_400HZ,
+    };
+    ESP_ERROR_CHECK(icm42670_config(handle, &imu_cfg));
+
+    return handle;
```

Comment:
`handle` is of type `icm42670_handle_t` but return type is `bsp_imu_t`. How do you want to keep this in sync?

Reply:
I added cast to return. Is it ok?